src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProviderFactory.cs

```diff
@@ -155,6 +165,11 @@ private AzureStorageDurabilityProvider GetAzureStorageStorageProvider(DurableCli
                 UseLegacyPartitionManagement = this.azureStorageOptions.UseLegacyPartitionManagement,
             };
 
+            if (this.inConsumption)
```

Comment:
I'm guessing the reason we do this here as opposed to above is because this is not exposed on the host.json?

It's also worth noting that I'm not sure this is any different that the default value for this setting anyways.

Reply:
I think you're right. It's `5 * ProcessorCount` which I think is 1 in the consumption plan.

https://github.com/Azure/durabletask/blob/d3c53b69259ea401839a8dd2aa21f96857a79314/src/DurableTask.AzureStorage/AzureStorageOrchestrationServiceSettings.cs#L114

I just went off the literal recommendation here: https://github.com/Azure/azure-functions-durable-extension/issues/1646

@cgillum, any thoughts on whether or not this setting's default value should be tweaked?
Another alternative is that I can leave it as-is and simply expose it for customers to tweak

Reply:
Consumption VMs on Windows can have as many as 4 vCPUs but the app is only allowed to use one of them, which is why we need special handling (in other words, I expecte `Environment.ProcessorCount` may return 4 and therefore cause us to increase network concurrency more than we want to.